rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -2734,10 +2747,28 @@ RMW_Connext_Client::send_request(
     return RMW_RET_ERROR;
   }
 
+#ifndef TRACETOOLS_DISABLED
```

Comment:
same here, we could call `rmw_api_connextdds_get_gid_for_client` to fetch the client gid and add it to tracepoint information?

Reply:
see https://github.com/ros2/ros2_tracing/pull/145#discussion_r1876858777